brownie-config.yaml

```diff
@@ -17,12 +17,20 @@ networks:
     nu_token_supply: 1_000_000_000
     t_staking: '0x18eFb520dA5D387982C860a64855C14C0AcADF3F'
     work_lock: '0x0000000000000000000000000000000000000000'
+    staking_escrow: '0x6A6F917a3FF3d33d26BB4743140F205486cD6B4B'
+    pre_min_authorization: 40000000000000000000000
+    pre_min_operator_seconds: 86400  # one day in seconds
     verify: True
   mainnet:
     nu_token: '0x4fE83213D56308330EC302a8BD641f1d0113A4Cc'
     t_staking: '0x01B67b1194C75264d06F808A921228a95C765dd7'
     work_lock: '0xe9778E69a961e64d3cdBB34CF6778281d34667c2'
+    staking_escrow: '0xbbD3C0C794F40c4f993B03F65343aCC6fcfCb2e2'
+    pre_min_authorization: 40000000000000000000000
+    pre_min_operator_seconds: 86400  # one day in seconds
     verify: True
   development:
     nu_token_supply: 1_000_000_000
+    pre_min_authorization: 40000000000000000000000
+    pre_min_operator_seconds: 86400  # one day in seconds
     verify: False
```

Comment:
```suggestion
    verify: False

```

Reply:
why remove these? they are required by the constructor, no?